plugin/storage/badger/spanstore/read_write_test.go

```diff
@@ -244,7 +258,9 @@ func TestIndexSeeks(t *testing.T) {
 
 		trs, err = sr.FindTraces(context.Background(), params)
 		assert.NoError(t, err)
-		assert.Equal(t, 6, len(trs))
+		assert.Equal(t, 5, len(trs))
```

Comment:
5 is the correct number, there was an empty TraceID element in the previous version because the check for prevTraceID in the loop did not account the first iteration correctly.